doc/release.md

```diff
@@ -159,20 +162,28 @@ hub release download v${new_version}
 ```
 or download them manually.
 
-Then, upload them to PyPI like so:
+Once you have them downloaded, upload them to PyPI like so:
 ```
 twine upload *.whl
 ```
 twine will correctly determine the version from the filenames.
 
-4\. Publish the new release on PyPI (requires an authorized account).
+4\. Once the wheels are uploaded, publish the new release on PyPI (requires an authorized account).
 ```
 make dist
 twine upload dist/sourmash-${new_version}.tar.gz
 ```
 
-5\. Add the release on GitHub, using the tag you just pushed.
-Name it 'version X.Y.Z', and copy and paste in the release notes.
+(This should be done *after* the wheels are available, because some of
+the conda package build steps require the wheels and are automatically
```

Comment:
```suggestion
the conda package build steps require the source dist and are automatically
```

Reply:
I don't think this is correct tho? per https://github.com/dib-lab/sourmash/issues/874#issuecomment-578968033 when you trigger the bioconda package build with the update to PyPI, it fails. I thought this was b/c the wheels aren't available. Is it because sourmash-minimal isn't available?